releasenotes/notes/fix-tests-failing-requirements-8b5124a3336d1297.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
```

Comment:
I don't know if we need this release note. Or at least we should rewrite it as we remove the constraints on matplotlib and aer since both should be fixed before the next qiskit_experiments release.

Reply:
We could always remove the release note and add one later once the fix for both is out.

Reply:
Resolved by 5eba853.